modules/migration-converting-the-storage-class-of-persistent-volumes-cam.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-converting-the-storage-class-of-persistent-volumes-cam_{context}"]
+= Converting the storage class of persistent volumes in the {mtc-short} web console
+
+You can convert the storage class of persistent volumes (PVs) by migrating them within the same cluster. For that, you must create and run a migration plan in the {mtc-full} ({mtc-short}) web console.
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on the cluster on which {mtc-short} is running.
+* You must add the cluster to the {mtc-short} web console.
+
+.Procedure
+
+. In the left-side navigation pane of the web console, click *Projects*.
+. In the list of projects, click your project.
++
+The *Project details* page opens.
+. Click the DeploymentConfig. Note the name of its running pod.
+. Open the YAML tab of the project. Find the PVs and note the names of their corresponding persistent volume claims (PVCs).
+. In the {mtc-short} web console, click *Migration plans*.
+. Click *Add migration plan*.
+. Enter the *Plan name*.
++
+The migration plan name must not exceed 253 lower-case alphanumeric characters (`a-z, 0-9`) and must not contain spaces or underscores (`_`).
+
+. In *Migration type*, select *Storage class conversion*.
+. In *Source cluster*, select *host*.
+. Click *Next*.
++
+The *Namespaces* page opens.
+. Select the required project.
+. Click *Next*.
++
+The *Persistent volumes* page opens. The page displays the PVs in the project, all selected by default.
+. For each PV, select the desired target storage class.
+. Click *Next*.
++
+The wizard validates the new migration plan and shows that it is ready.
+. Click *Close*.
++
+The new plan appears on the *Migration plans* page.
+. To start the conversion, click the three dots menu of the new plan and select *Migrations*, then *Cutover*.
```

Comment:
So, actually there are 2 options

Stage & Cutover

Shall we mention the description of both and ask the user to choose appropriately?

Reply:
@stillalearner Is the Stage option relevant for storageclass conversion?

Reply:
@sbeskin-redhat  yes, it is. Here are the details of both migrations. Stage and Cutover

During a stage migration:
PV data is copied to the converted PVs.
PVC references in the applications are **not** updated to new PVCs.

During a cutover migration:
All transactions on source applications are halted for the duration of the migration.
PVC references in the applications are updated to new PVCs before restarting the applications.

Reply:
@stillalearner  So what are the considerations for choosing one of those options? And if the user chooses Stage, how does the migration proceeds?